Intro Cutscene: Improvements

Rusty-Box opened the pull request:
- Update intro.nut (so it works with the cutscene)
- Make the intro look more like a cutscene, instead of a level

Rusty-Box committed:
Rework Intro Cutscene

-Make the intro look more like a cutscene instead of a level
-Update intro.nut

qwertychouskie commented on levels/world1/intro.nut:
```diff
@@ -207,13 +217,16 @@ function shake_bush_thread(table)
   table.shake_bush();
   table.wait(3);
   table.shake_bush();
+  Text.set_text(_("Suddenly, Nolok jumped from behind\nof an ice bush!"));
```
That should be "Suddenly, Nolok jumped out from behind\nan ice bush!"

qwertychouskie commented on levels/world1/intro.nut:
```diff
@@ -170,21 +170,31 @@ function rap_scene()
+  wait(1.5);
+  Text.fade_out(0.5);
+  wait(0.3);
+  Text.set_text(_("Tux: Oh no. Nolok must have kidnapped her.")); // t=6.9
```
Maybe "Tux: Oh no... Nolok must have kidnapped her!"

maxteufel commented:
Squash the commits. And don't always say 'Improvements', it gets boring :smile:.

tobbi commented:
At the beginning I can see Tux falling on camera. Can we fix that somehow? Maybe hide tux first?

tobbi commented:
Also, slight suggestion: After asking "Where are you, Penny?" Tux could maybe do searching motions? Not sure if that can be shown, though.

Rusty-Box commented:
@tobbi Weird... . When I was testing it, you couldn't see him at the beginning. And I also tired to make a search motion. ~~But it never worked the way it should be.~~

Karkus476 commented:
@Rusty-Box Make tux look left right left right in quick succession maybe?

Rusty-Box committed:
Update intro.stl

Rusty-Box committed:
Update intro.nut

- Make tux look left and right, when looking for penny
- fix some scripts
- make logo fade in and out instead of make it appear instantly

Rusty-Box commented:
I think it's almost done. Only problem is that Tux isn't invisible at the start of the cutscene, when starting it in the main game. (But it works when starting it in the level editor)
